# ap_dict_message: '{"reach(object_69)": "p1", "reach(object_62)": "p2", "reach(object_72)": "p3"}'
# ap_dict_message: '{"reach(object_5)": "p0", "reach(object_6)": "p1"}'
ap_dict_message: '{"reach(object_3)": "p0", "reach(object_1)": "p1"}'